runtime/dotnet/azurefunctions/Startup.cs

```diff
@@ -43,19 +43,16 @@ private IConfigurationRoot BuildConfiguration(string rootDirectory)
             var config = new ConfigurationBuilder();
 
             // Config precedence 1: root app.settings
-            config
-                .SetBasePath(rootDirectory)
-                .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
-                .UseLuisConfigAdapter()
-                .UseLuisSettings();
+            config.SetBasePath(rootDirectory);
+            config.AddJsonFile("appsettings.json", optional: true, reloadOnChange: true);
```

Comment:
What is this additional appsettings.json file for?

Why do we need it as well as the one in settings/appsettings.json?

Reply:
remove the appsettings.json